src/lib/services/version.service.ts

```
@@ -138,6 +138,31 @@ export class VersionService {
+    await prisma.version.updateMany({
+      where: { promptId: version.promptId },
+      data: { isActive: false },
+    })
+
+    // Activate the selected version
+    const updatedVersion = await prisma.version.update({
+      where: { id: versionId },
+      data: { isActive: true },
+    })
+
+    return this.mapVersion(updatedVersion)
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)